core/src/main/java/hudson/model/AbstractProject.java

```diff
@@ -1922,9 +1922,8 @@ public DirectoryBrowserSupport doWs( StaplerRequest req, StaplerResponse rsp ) t
      */
     public HttpResponse doDoWipeOutWorkspace() throws IOException, ServletException, InterruptedException {
         checkPermission(Functions.isWipeOutPermissionEnabled() ? WIPEOUT : BUILD);
-        R b = getSomeBuildWithWorkspace();
-        FilePath ws = b!=null ? b.getWorkspace() : null;
-        if (ws!=null && getScm().processWorkspaceBeforeDeletion(this, ws, b.getBuiltOn())) {
```

Comment:
I see the method as deprecated. Am I missing something here. Please correct me if I am wrong
https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/scm/SCM.java#L223